FirebaseCombineSwift/Sources/Firestore/CollectionReference+Combine.swift

```diff
@@ -0,0 +1,87 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#if canImport(Combine) && swift(>=5.0) && canImport(FirebaseFirestore)
```

Comment:
Why do we need to check `canImport(FirebaseFirestore)`? Won't the CocoaPods or SwiftPM dependency make sure it is there?

Reply:
I think these are still there from the initial setup in which we wanted to try to use a single umbrella module that would only expose the extensions for the modules that the user had imported as well.